sonic-net/sonic-utilities
# clear: Fix clear queuecounters to also clear VOQ counters

#### What I did

Fix a bug where the CLI command `sonic-clear queuecounters` would not clear the VOQ counters.

fixes sonic-net/sonic-buildimage#15198

#### How I did it

Added the command to clear the VOQ counters.

#### How to verify it

1. Start traffic on an interface on a specific queue.
2. Use `show queue counters` and `show queue counters --voq` to verify that the counters are non-zero.
3. Stop traffic on the interface.
4. Run `sonic-clear queuecounters`.
5. Repeat the show commands and ensure that the counters have been reset.

#### Previous command output (if the output of a command-line utility has changed)

#### New command output (if the output of a command-line utility has changed)
